source/draw/RasterPacked.ooc

```diff
@@ -89,4 +89,18 @@ RasterPacked: abstract class extends RasterImage {
 		file free()
 		StbImage writePng(filename, this size width, this size height, this bytesPerPixel, this buffer pointer, this size width * this bytesPerPixel)
 	}
+	swapChannels: func (first, second: Int) {
+		version(safe) {
+			if (first > this bytesPerPixel || second > this bytesPerPixel)
+				raise("Channel number too large")
+		}
+		pointer := this buffer pointer
+		index := 0
+		while (index < this buffer size) {
+			value := pointer[index + first]
+			pointer[index + first] = pointer[index + second]
+			pointer[index + second] = value
+			index += this bytesPerPixel
```

Comment:
Should we have an empty `swapChannels: override func (first, second: Int)` in `RasterMonochrome` ? Or do we trust that no one will ever want to swap channels on an image with single channel ? :)